pkg/controllers/securitypolicy_controller.go

```diff
@@ -27,13 +28,35 @@ import (
 	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-var (
-	log = logf.Log.WithName("controller").WithName("securitypolicy")
-)
-
 const (
 	WCP_SYSTEM_RESOURCE = "vmware-system-shared-t1"
 	METRIC_RES_TYPE     = "securitypolicy"
+	MAX_WORKER_COUNT    = 5
```

Comment:
is it possible to get runtime cpu number and calculate the MAX_WORKER_COUNT?

Reply:
Good idea